components/Buttons/tests/unit/ButtonsTests.m

```diff
@@ -1116,6 +1120,45 @@ - (void)testAdjustsFontProperty {
                              @"Font size should be equal to MDCFontTextStyleButton's.");
 }
 
+/**
+ Test legacy dynamic type has no impact when @c
+ adjustFontForContentSizeCategoryWhenScaledFontIsUnavailable is set to @c NO that the font stays
+ the same.
+ */
+- (void)testLegacyDynamicTypeDisabled {
+  // Given
+  UIFont *fakeFont = [UIFont systemFontOfSize:55];
+  [self.button setTitleFont:fakeFont forState:UIControlStateNormal];
+  UIFont *originalFont = self.button.titleLabel.font;
+  self.button.mdc_adjustsFontForContentSizeCategory = YES;
+
+  // When
+  self.button.adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable = NO;
+  [self.button updateTitleFont];
+
+  // Given
+  XCTAssertTrue([self.button.titleLabel.font mdc_isSimplyEqual:originalFont]);
+}
+
+/**
+ Test legacy dynamic type has no impact when @c
```

Comment:
This comment appears to be out of date; Dynamic Type does have an impact here.

Reply:
Whoops, copy and paste got me!